tests/testnet.spec.js

```diff
@@ -502,7 +507,7 @@ describe('Testing generic actions', () => {
       fioPublicKey: publicKey2
     })
 
-    expect(result).to.have.all.keys('balance')
+    expect(result).to.have.all.keys('balance','available')
     expect(result.balance).to.be.a('number')
```

Comment:
add `expect(result.available).to.be.a('number')`

Reply:
done